### PR TITLE
[cli] do not pass user-controlled input as format string

### PR DESCRIPTION
**Pull request by jwhui**

### CODE REVIEW

**Review by abtink — APPROVED** [review]
LGTM. Thanks for fixing it.

File: src/cli/cli.cpp [hunk 1]
```diff
@@ -584,7 +584,7 @@ otError Interpreter::ProcessDomainName(uint8_t aArgsLength, char *aArgs[])
 
     if (aArgsLength == 0)
     {
-        OutputLine(otThreadGetDomainName(mInstance));
+        OutputLine("s", otThreadGetDomainName(mInstance));
```
Comment (bukepo) [inline src/cli/cli.cpp:587]:
```suggestion
        OutputLine("%s", otThreadGetDomainName(mInstance));
```

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Pull request merged**